.github/renovate.json5

```
@@ -1,6 +1,10 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:base",
+  "extends": ["github>statnett/renovate-config"],
+  "packageRules": [
+    {
+      "matchUpdateTypes": ["minor", "patch", "digest"],
+      "automerge": true,
+    },
```

Currently invalid json, but might be alright (given that its json5?)
```suggestion
    }
```

---

LOL, this is the NICE thing with JSON5.